### ISSUE TITLE
USE NEW FORK - I don't think the master is supported anymore

### ISSUE THREAD

**Issue by rolinger** [EdwinHoksberg/php-fcm#15]
It doesn't look like Edwin has any interest in maintaining the master anymore.  I will probably create a new project off of my current fork to be able to properly support it for the future.  Until then I suggest using my fork as it has plenty of fixes and new features:

https://github.com/rolinger/php-fcm

**Comment by AndyGaskell** [comment]
Hi @rolinger 

Yea, I agree, Edwin hasn't made any changes for 7 months, so is perhaps no-longer keen to maintain this.

I've been using the code from your fork, and it seems great.  Were you still thinking of creating a new project from your fork?

I have some feedback on the forked code if you like, happy to share it.  

Thanks for the work on the fork, some great new features in there.

**Comment by rolinger** [comment]
@AndyGaskell - I was hoping he would merge the fork and maintain but he hasn't nor has he really ever responded.  We could create my fork as a new project and we could both be collaborators on it.  I added a manual fix for the deviceID (See issue #17) lookup that turns out is an option in guzzlehttp.

**Comment by AndyGaskell** [comment]
Hi @rolinger 

Yea, I've been working with your fork, and made hit a couple of minor issues that I've fixed locally.  It'd be good to have a way to move this forward and get your enhancements into a master. Also be good to be able to merge my minor fixes too.

If you're concerned about having the new project under your name, we could create a GitHub organisation with members.  I've found previously that can be a better way to manage code ownership, as it makes it less dependent on a single person.  I think it can also encourage more people to contribute.  It'd be easy to create a GitHub organisation called "php-fcm", for example and create the new project under that.

Thanks

**Comment by rolinger** [comment]
@andyGaskell - I am good with that....Give me a few days to set things up and I will invite you to the project.  I tried doing this in the past for a private project and ran into some issues and gave up.  So I will need to brush up on how to do this correctly.

**Comment by EdwinHoksberg** [comment]
Hi guys, sorry for the late replies! I don't use this project in production anymore so it has lost my focus. I'd be glad to transfer this project to either one of you. 
I could add you both as collaborators here or if you want to create a new project I could archive this one. Any preferences?

**Comment by AndyGaskell** [comment]
Hi @EdwinHoksberg 

Thanks for the reply, glad you're ok.  I always end up wondering, with repos that are no longer maintained, what has happened to the maintainer.

There is definitely an advantage to keeping the package in the same place, so adding me and @rolinger to this one would be a good solution.  Maintaining libraries is tricky for one person, I'm sure it'd be a positive thing if there were 2 or 3 people who could do merges and admin tasks.

Also, the CI/CD stuff is all nicely set-up at the moment.

Thanks for the excellent library by the way.  I looked at a few different options for Firebase and PHP, but this was by far the best, simple, clean and nice to use.

**Comment by EdwinHoksberg** [comment]
Thank you for your kind words @AndyGaskell! I've added you and @rolinger as collaborators to this project, you should've received an email with the invitation.

**Comment by AndyGaskell** [comment]
Thanks @EdwinHoksberg I've received and accepted the github invite, that's all working well.  There was a small PR for a doc change, so I merged that in, to check it's all working ok.

Moving forward, @rolinger and me will work on getting his new features into the master and testing it all.

At my work we've just integrated it into our production platform, so should be working with it for a couple of years at least.  So I'll be very happy to help look after it, and keep it updated and maintained.
